dev_guide/new_app.adoc

```diff
@@ -483,6 +483,37 @@ from the global library:
 ====
 
 image::console_select_image.png["Select Builder Image"]
+
+[NOTE]
+Only ImageStreamTags which have the 'builder' tag listed in their annotations will appear in this list, as demonstrated here:
+
+----
+    {
+      "kind": "ImageStream",
+      "apiVersion": "v1beta3",
+      "metadata": {
+        "name": "ruby",
+        "creationTimestamp": null
+      },
+      "spec": {
+        "dockerImageRepository": "registry.access.redhat.com/openshift3/ruby-20-rhel7",
+        "tags": [
+          {
+            "name": "2.0",
+            "annotations": {
+              "description": "Build and run Ruby 2.0 applications",
+              "iconClass": "icon-ruby",
```

Comment:
Would this be the appropriate place to explain the effect of the other annotations like iconClass ?